### PR TITLE
feature: Added signup page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added E-Commerce Sign up page

Live Link: https://neog-e-comm-website.netlify.app/signup/index.html

### CODE REVIEW

File: signup/index.html [hunk 1]
```diff
@@ -19,23 +19,23 @@
...
 
             <div class="nav-right-container">
-                <button class="btn btn-primary">Login</button>
-                <i class="far fa-heart fa-lg"></i><span class="badge1">4</span>
-                <i class="fas fa-shopping-cart fa-lg"></i><span class="badge2">4</span>
+                <button class="login-btn">Login</button>
+                <i class="far fa-heart fa-lg"></i>
+                <i class="fas fa-shopping-cart fa-lg"><span class="badge2">4</span></i>
             </div>
         </nav>
```
Comment (tushargoyal812) [inline signup/index.html:36]: suggestion:remove unnecessary space

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge just take care of suggestion.

File: signup/index.html (hunk 1 above)
Comment (poojakelkar) [reply signup/index.html:36]: sure Thanks @tushargoyal812

**Review by akash-1318 — COMMENTED** [review]
Looks good after all the changes you have done so you can continue merging this branch.

**Comment by poojakelkar** [comment]
Thank you @akash-1318

**Pull request merged**